test/unit/Vault/Undercollateralization.t.sol

```diff
@@ -213,7 +214,7 @@ contract VaultUndercollateralizationTest is UnitBaseSetup {
     uint256 _yieldFeeShares = vault.yieldFeeTotalSupply();
 
     vm.expectRevert(abi.encodeWithSelector(VaultUnderCollateralized.selector));
-    vault.mintYieldFee(_yieldFeeShares, address(this));
+    vault.mintYieldFee(_yieldFeeShares);
```

Comment:
Should also assert that bob received his shares below, so that the test would fail otherwise.

Reply:
The Vault is now undercollateralized, so any yield fee that has not been claimed will be shared proportionally amongst depositors.
Here we test that the call to `mintYieldFee` reverts since we should not be able to mint yield fee shares.